README.md

```diff
@@ -0,0 +1,201 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> # Finance Tracker
+
+| Project Login Page |
+|---------------------------------------|
+|<div align="center" width="auto"><img alt="Finance-Tracker login" src="app/assets/images/Finance-Tracker login.PNG"/></div>|
+
+
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+   - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Finance Tracker] <a name="about-project"></a>
+
+> **[Finance-Tracker]** The Ruby on Rails Budget-app project is aimed at creating a mobile web application that enables budget management, including user registration and login for data privacy, the introduction of new transactions linked to categories, and the display of expenditures per category.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp">CSS</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Splash screen]**
+- **[Sign up and log in pages]**
+- **[Home page (categories page)]**
+- **[Details page (transactions page)]**
+- **["Add a new category" page]**
+- **["Add a new transaction" page]**
+
+| Implemented Schema's ERD |
+|---------------------------------------|
+|<img width="auto" alt="erd_diagram" src="app/assets/images/erd_diagram.png">|
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo & Project intro <a name="live-demo"></a>
+ - [Live Demo on Render](https://finance-tracker-kxhn.onrender.com/)
+ - [Introduction to the project](https://www.loom.com/share/928b45d5bd4c45d4bc95497e9ccc442f?sid=4e1fcb03-dfd3-4891-b610-0ff3c4545693)
```

Comment:
- [ ] [REQUIRED] Great presentation @ab-noori 🥇 , however, a part of the project requirement states that you should highlight a piece of the code that you are most proud of. Kindly implement this before requesting another review 👍🏽